SPECS/tdnf/tdnf.spec

```diff
@@ -219,6 +219,9 @@ fi
 /%{_lib}/systemd/system/tdnf*
 
 %changelog
+* Thu Aug 29 2024 Sam Meluch <sammeluch@microsoft.com> - 3.5.6-3
+- Add virtual repo snapshot exclude
```

Comment:
Is there a reason we want a separate argument for the snapshot instead of relying on the existing `--disablerepo` and `--enablerepo`? I'm worried that having two may be confusing when using the snapshot.

Reply:
The reason to add a snapshot exclude option is to allow for a repo containing new packages locally built to still be included when building an image, but not from upstream. This is also additional flexibility within the feature for tdnf, not just applicable to our use case for allowing local RPMs from within the toolkit.